thrift/ThriftTest.thrift

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright 2022 Meta
```

Comment:
The earliest public commit that I can find for this is here:

https://svn.apache.org/viewvc/incubator/thrift/trunk/test/ThriftTest.thrift?view=markup&pathrev=664712
https://thrift.apache.org/static/files/thrift-20070401.pdf
https://mcslee.com/#code

Mark C. Slee was the original author of Thrift, so this should probably say

`Copyright 2006 Facebook`